### PR TITLE
[DEV-1080] Webinar questions form

### PR DESCRIPTION
**Pull request by jeremygordillo**
#### Motivation and Context

We need a form where webinar registrants can submit questions

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      </Typography>
+
+      <TextField
+        disabled={disabled}
+        label={t('questionsForm.question')}
+        multiline
+        maxRows={6}
+        value={question}
+        variant='outlined'
+        onChange={handleChange}
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx:74]:
```suggestion
        onChange={handleChange}
        sx={{ flexGrow: 1 }}
```

Comment (marcobottaro) [reply apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx:74]: The result will be like this:
image

mobile:
image

File: apps/nextjs-website/src/components/organisms/WebinarDetailTemplate/WebinarDetailTemplate.tsx [hunk 2]
```diff
@@ -54,6 +55,7 @@ const WebinarDetailTemplate = ({ webinar }: WebinarDetailTemplateProps) => {
         endDateTime={webinar.endDateTime}
       >
         {subscribeToWebinarButton}
+        <WebinarQuestionsForm />
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/organisms/WebinarDetailTemplate/WebinarDetailTemplate.tsx:58]: You can remove this temporary line (and the import at line 16)

**Review by marcobottaro — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

File: apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { Done } from '@mui/icons-material';
+import { LoadingButton } from '@mui/lab';
+import { Card, TextField, Typography } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { useState } from 'react';
+
+const sendQuestion = (question: string) => {
+  return new Promise((resolve) => {
+    console.log(question);
```
Comment (datalek) [inline apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx:9]:
```suggestion
```

**Review by datalek — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Review by datalek — APPROVED** [review]

**Pull request merged**